spring-ai-spring-boot-autoconfigure/src/main/java/org/springframework/ai/autoconfigure/chat/client/ChatClientBuilderProperties.java

```diff
@@ -36,12 +36,48 @@ public class ChatClientBuilderProperties {
 	 */
 	private boolean enabled = true;
 
+	private Observation observation = new Observation();
+
+	public Observation getObservation() {
+		return this.observation;
+	}
+
 	public boolean isEnabled() {
-		return enabled;
+		return this.enabled;
 	}
 
 	public void setEnabled(boolean enabled) {
 		this.enabled = enabled;
 	}
 
+	public static class Observation {
+
+		/**
+		 * Whether to include the input content in the observations.
+		 */
+		private boolean includeInput = false;
```

Comment:
Can we reuse the same property we have already for the low-level model observations, and control in one place whether we want the print out the prompt content?

Reply:
Thought about it but my feeling is that we should not. the low level controls all chat models while the chat client is one. Furthermore it is nice to keep more fine grain control. 
We can change this in the future if we find it useful to control everything with a single property.